src/pool-bin/BinPositionManager.sol

```diff
@@ -240,11 +238,8 @@ contract BinPositionManager is
             ZERO_BYTES
         );
 
-        // delta amt0/amt1 will either be 0 or positive in removing liquidity
-        if (delta.amount0() < 0 || delta.amount1() < 0) revert IncorrectOutputAmount();
-        if (uint128(delta.amount0()) < params.amount0Min || uint128(delta.amount1()) < params.amount1Min) {
-            revert OutputAmountSlippage();
```

Comment:
I guess we can remove the event definition as well

Reply:
Not used elsewhere

![CleanShot 2024-09-19 at 16 24 19@2x](https://github.com/user-attachments/assets/bb12727d-a8af-4f07-883f-217b20751698)

Reply:
removed